seed/utils/sensors.py

```diff
@@ -78,9 +77,7 @@ def _usages_by_exact_times(self, page, per_page):
                 for sensor in self.sensors
             }
 
-            sensor_readings = SensorReading.objects.filter(timestamp__range=[earliest_time, latest_time], sensor__in=self.sensors)
-            if self.showOnlyOccupiedReadings:
-                sensor_readings = sensor_readings.filter(is_occupied=True)
```

Comment:
Why isn't this needed anymore?

Reply:
It's already handled at the top, no need to do it twice